spec/models/trip_spec.rb

```diff
@@ -0,0 +1,93 @@
+require 'rails_helper'
+
+describe Trip, type: :model do
+
+  before(:all) do
+    city = City.create!
+
+    @trip = Trip.create!(
+      departure_date: Date.today,
```

Comment:
Codacy suggest Time.zone.today

Reply:
Ok it doesn't matter in test I update it